### ISSUE TITLE
petsc BuildSystem reference doesn't seem to exist

### ISSUE THREAD

**Issue by jloveric** [lanl/jali#17]
I'm trying to run the config/configure.py command I get this 

    ++ Could not locate BuildSystem in /***/jali/config. 
    ++ Downloading it from http://petsc.cs.iit.edu/petsc/BuildSystem.
    ++ Attempting: hg clone http://petsc.cs.iit.edu/petsc/BuildSystem 
    ++ Unable to clone BuildSystem.  Attempting download http://petsc.cs.iit.edu/petsc/BuildSystem/archive/tip.tar.gz

Could be the site is down or that the BuildSystem was moved. 

I've found another one here, but haven't tried it https://github.com/erdc/petsc-BuildSystem

**Comment by raovgarimella** [comment]
Jali should not be configured using the configure.py script. That is an internal file. It should be built using CMake as shown in the README.md file

**Comment by jloveric** [comment]
Yes, so I need all the tpls.  I'm trying to build on Ubuntu and then create a Dockerfile so the complete installation procedure can just be copied (installing all dependencies etc...).

**Comment by raovgarimella** [comment]
There are instructions on how to build the TPLs using the SuperBuild script.
